src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js

```diff
@@ -79,7 +99,7 @@ export default View.extend({
             {{#if sameDeviceOVEnrollmentEnabled}}
               <li>
                 <span>{{i18n code="customUri.required.content.download.title" bundle="login"}}</span>&nbsp;
-                <a href="{{deviceMap.downloadHref}}" target="_blank" id="download-ov" class="orOnMobileLink">
```

Comment:
this was causing the download link to navigate to the enroll on another device flow

Reply:
Can you point out where this unwanted behavior was coming from?

Reply:
yeah [here](https://github.com/okta/okta-signin-widget/blob/d314a99c3b0344f94e0144185404d004d2b60a57/src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js#L196) is where the click event is attached to the Set up on another device link and looks for this classname.  Since this download link was using the same classname, that event was being triggered when this link was pressed.